### ISSUE TITLE
The basic Demo style is out of order and cannot be entered and displayed normally

### ISSUE THREAD

**Issue by aoxiangsky** [nhn/tui.editor#2072]
![1638401937_1_.png](https://i.loli.net/2021/12/02/HPFUDZOdtGj3YpJ.png)

``` javascript
import '@toast-ui/editor/dist/toastui-editor.css';
import { Editor } from '@toast-ui/react-editor';
import Box from '@mui/material/Box';

function ArticleEdit() {
    return (
        <Box
            sx={{
                p: 2,
                backgroundColor: 'common.white',
                minHeight: '75%',
            }}
        >
            <Editor
                initialValue="Text here!"
                previewStyle="vertical"
                height="auto"
                initialEditType="markdown"
                useCommandShortcut={true}
                // plugins={[colorSyntax]}
                // plugins={[chart, codeSyntaxHighlight, uml]}
            />
        </Box>
    );
}

export default ArticleEdit;
```

Is it where I am missing a configuration item?

**Comment by js87zz** [comment]
@aoxiangsky 
Can you give me your code in code sandbox?(included #2074, #2075 )

**Comment by aoxiangsky** [comment]
I solved it, just upgrade the package to 3.x version. NPM downloads the 2.x package by default .
